Polyfills/Window/Source/Window.cpp

```diff
@@ -10,18 +11,23 @@ namespace Babylon::Polyfills::Internal
         constexpr auto JS_CLASS_NAME = "Window";
         constexpr auto JS_SET_TIMEOUT_NAME = "setTimeout";
         constexpr auto JS_A_TO_B_NAME = "atob";
+
         constexpr auto JS_ADD_EVENT_LISTENER_NAME = "addEventListener";
         constexpr auto JS_REMOVE_EVENT_LISTENER_NAME = "removeEventListener";
+        auto JS_DEVICE_PIXEL_RATIO_NAME = "devicePixelRatio";
     }
 
-    void Window::Initialize(Napi::Env env)
+    void Window::Initialize(Napi::Env env, void* windowPtr)
     {
         Napi::HandleScope scope{env};
 
-        Napi::Function constructor = DefineClass(
+        Napi::Function constructor = Window::DefineClass(
             env,
             JS_CLASS_NAME,
-            {});
+            {
+                StaticAccessor(JS_DEVICE_PIXEL_RATIO_NAME, &Window::GetDevicePixelRatio, &Window::SetDevicePixelRatio)
```

Comment:
Seems like this should be an `InstanceAccessor`. `GetDevicePixelRatio` is on the global `window` instance, not on the prototype.

Reply:
I'll try that but I don't believe this will work either.

Reply:
@bghgary helped me find a solution, since the javascript `Window` class we create here isn't actually the `window` reference we use in our environment. the following `Set()` declarations adds the other polyfill methods to the global object [which we set to also be global\["window"\]](https://github.com/BabylonJS/BabylonNative/blob/93005ea71a30f489eac3ad1d98878ac50532b51a/Core/JsRuntime/Source/JsRuntime.cpp#L20)

Manually adding an accessor for window .polyfill seems to give us comparable behavior to the window object in a browser.